runtime/kreivo/src/communities/memberships.rs

```diff
@@ -0,0 +1,61 @@
+use super::*;
+
+use frame_support::traits::EitherOf;
+use frame_system::EnsureRootWithSuccess;
+use virto_common::MembershipId;
+
+pub type CommunityMembershipsInstance = pallet_nfts::Instance2;
+
+// From https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs#L810
+impl pallet_nfts::Config<CommunityMembershipsInstance> for Runtime {
```

Comment:
That said, perhaps it's worth changing the membership traits to be implemented over nonfungibles instead of nonfungible it can help us with some problems we have like counting the members of a community and gives some order to have each community with its own set of memberships.

Reply:
We've resolved removing the `Instance1` for nfts instead, as it's not currently usable in Kreivo.